web/src/components/footer.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import AIRtecAwhite from './svgs/AIRtecAwhite';
+
+export default function Footer() {
+    const externalLinks = [
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+    ];
+    const internalLinks = [
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+    ];
+    return (
+        <footer className="bg-brand-1 text-ts-body w-full h-96 text-white font-normal fixed bottom-0">
+            <div className="flex justify-between container pt-11">
```

Comment:
Small thing, I like to alphabetize these as much as possible. That _usually_ puts `container` in the front (or close to it). I think this is helpful visually (especially when I'm doing reviews ;-) )